dot/sync/chain_sync_integeration_test.go

```diff
@@ -40,8 +43,8 @@ func newTestChainSync(t *testing.T) (*chainSync, *blockQueue) {
 	bs.On("HasHeader", mock.AnythingOfType("common.Hash")).Return(true, nil)
 
 	net := new(syncmocks.Network)
-	net.On("DoBlockRequest", mock.AnythingOfType("peer.ID"),
-		mock.AnythingOfType("*network.BlockRequestMessage")).Return(nil, nil)
+	net.On("DoBlockRequest", mock.AnythingOfType("peer.ID"), mock.AnythingOfType("*network.BlockRequestMessage")).
+		Return(nil, nil)
```

Comment:
nit to keep it as close as it was for git file moves tracking

```suggestion
	net.On("DoBlockRequest", mock.AnythingOfType("peer.ID"), 
		mock.AnythingOfType("*network.BlockRequestMessage")).Return(nil, nil)
```

Reply:
Updated.